cortex/cli.py

```
@@ -15,18 +17,25 @@
+    and package conflict resolution capabilities.
+    """
+    
     def __init__(self):
         self.spinner_chars = ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏']
         self.spinner_idx = 0
-        self.prefs_manager = None  # Lazy initialization
+        self.prefs_manager = PreferencesManager()
+        try:
+            self.prefs_manager.load()
+        except Exception:
+            pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)